src/credit.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+function Credit (account) {
```

Comment:
Is `Credit` really a "thing" (a domain object)? It feels more like a way to make a credit on an account.